exampleCourse/questions/element/matching/question.html

```diff
@@ -150,3 +150,25 @@
         </pl-matching>
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-body">
+        <pl-question-panel>
+            <p>
+                Example 8: This example uses the "bottom" placement for the options. This is useful for questions with long statements and options
```

Comment:
```suggestion
                Example 8: This example uses the `options-placement="bottom"` argument. This is useful for questions with long statements and options, which may wrap over multiple lines or cause formatting issues in the default "right" placement.
```

Reply:
Oh, sorry about the typos. Definitely could have caught that before review. I've pushed the edits.